docs/troubleshooting.md

```diff
@@ -0,0 +1,24 @@
+# How to troubleshoot issues
+
+Sometimes Ollama may not perform as you expect. One of the best ways to figure out what just happened is to take a look at the logs. You can find the logs on Mac by running the command:
+
+```shell
+cat ~/.ollama/logs/server.log
+```
+
+On Linux systems with systemd, the logs can be found with this command:
+
+```shell
+journalctl -u ollama
+```
+
+If you are manually running `ollama serve` in a terminal, the logs will be on that terminal.
+
+If you need help interpreting what you see, try signing in to the Discord at [discord.gg/ollama](https://discord.gg/ollama).
+
+## Known issues
```

Comment:
This is a good spot to link to major known issues (e.g. ones we've closed repeated issues around)

Reply:
so not just explaining common messages in the logs

Reply:
A combo of error message + link to issue for further discussion would be best!